### PR TITLE
Add some more stuff

### PR DESCRIPTION
**Pull request by mezz**

### CODE REVIEW

**Comment by JamiesWhiteShirt** [comment]
It's not sorted alphanumerically, but then again, what's the point.

File: data/badverbinfinitive [hunk 1]
```diff
@@ -7,3 +7,4 @@ crash
 make the game too hard
 spam the log
 steal logins
+collect data
```
Comment (Gaelan) [inline data/badverbinfinitive:10]: please add things to both versions of badverb (inf. and present)

File: data/badverbpresent [hunk 2]
```diff
@@ -7,3 +7,4 @@ crashes
 makes the game too hard
 spams the log
 steals logins
+requires signing a EULA
```
Comment (Gaelan) [inline data/badverbpresent]: this too

File: data/modaction [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Blogging
+Tweeting
+Snacking
+Torturing
+Lawnmowing
+Cultivating
+Studying
+Parading around
+Exploring
+Painting
```
Comment (Gaelan) [inline data/modaction:47]: should these all be capitalized?

**Review by Gaelan — CHANGES_REQUESTED** [review]
generally pretty good

File: data/modaction (hunk 3 above)
Comment (mezz) [reply data/modaction:47]: Yes, they are capitalized like `modthing` and used in `modname`s.

**Comment by Gaelan** [comment]
can confirm, is fix

**Review by Gaelan — APPROVED** [review]

**Pull request merged**